include/RAJA/util/View.hpp

```diff
@@ -138,6 +143,45 @@ using TypedManagedArrayView = TypedViewBase<DataType,
 #endif
 
 
+template <typename ViewType,
+          typename AtomicPolicy = RAJA::auto_atomic>
+struct AtomicViewWrapper {
+  using Base = ViewType;
+  using DataPointer = typename Base::DataPointer;
+  using DataType = typename Base::DataType;
+  using AtomicType = RAJA::AtomicRef<DataType, AtomicPolicy>;
+
+  Base base_;
```

Comment:
i agree, but I was following the pattern used previously.  perhaps I'll change them all.